pkg/services/services.go

```diff
@@ -144,13 +143,10 @@ func getKubeletServiceConfiguration(argsFromIginition map[string]string, debug b
 		return servicescm.Service{}, err
 	}
 	var powershellVars []servicescm.PowershellCmdArg
-	hostnameArg := getKubeletHostnameOverride(platform)
+	hostnameArg, hostnamePowershellVars := getKubeletHostnameOverride(platform)
```

Comment:
nit: `hostnamePowershellVar` instead of the plural `hostnamePowershellVars`, I had to re-read to make sure `getKubeletHostnameOverride()` only return one PowershellCmdArg

Reply:
Addressed.